core/src/server/rpc_module.rs

```diff
@@ -504,6 +504,13 @@ impl<Context> RpcModule<Context> {
 	pub fn new(ctx: Context) -> Self {
 		Self { ctx: Arc::new(ctx), methods: Default::default() }
 	}
+
+	/// Transform a module into an `RpcModule<()>` (unit context).
+	pub fn decontextualize(self) -> RpcModule<()> {
```

Comment:
I found the method name a bit long to read, could `strip_context`, `remove_context` or something work?